### PR TITLE
Lint branding directory

### PR DESCRIPTION
**Pull request by mauriciovieira**
This PR is part of #236 endeavor.

### CODE REVIEW

**Comment by mauriciovieira** [comment]
@Digipalvelutehdas/apika-tiimi please review

**Comment by mauriciovieira** [comment]
@Digipalvelutehdas/apika-tiimi please review

**Comment by brylie** [comment]
On this branch, the *Branding Logo* preview is failing to render on the **Branding Settings** page:

screenshot_20170206_122932

**Comment by brylie** [comment]
@mauriciovieira, please also resolve conflicts.

File: branding/logo/client/view/view.js [hunk 1]
```diff
@@ -1,35 +1,43 @@
...
     const objectId = new Mongo.Collection.ObjectID(currentProjectLogoFileId);
 
     // Get project logo file Object
     const currentProjectLogoFile = ProjectLogo.findOne(objectId);
 
+    let projectLogoFileUrl;
     // Check if project logo file is available
     if (currentProjectLogoFile) {
-      // Get API logo file URL
-      return Meteor.absoluteUrl().slice(0, -1) + ProjectLogo.baseURL + "/md5/" + currentProjectLogoFile.md5;
```
Comment (brylie) [inline branding/logo/client/view/view.js]: It looks like this change introduced the regression on the **Branding Settings** logo preview. Our logo URL should include the `/md5/${currentProjectLogoFile.md5` in order to work properly.

Lets fix the regression, while keeping the explanatory variables where possible.

**Comment by mauriciovieira** [comment]
Fixing eslint.

**Comment by brylie** [comment]
OK, good work. Now I am getting an exception when trying to upload a logo:

```js
Exception in delivering result of invoking '/ProjectLogo.files/insert': TypeError: fileNameEndsWith is not a function
```

**Comment by brylie** [comment]
I fixed the issue by removing destructuring braces from the import.

**Comment by brylie** [comment]
This is ready to merge. @mauriciovieira please review my recent change and merge this PR.

**Pull request merged**